pkg/services/user/userimpl/user.go

```diff
@@ -486,7 +486,7 @@ func (s *Service) CreateServiceAccount(ctx context.Context, cmd *user.CreateUser
 
 	err = s.store.LoginConflict(ctx, cmd.Login, cmd.Email, s.cfg.CaseInsensitiveLogin)
```

Comment:
this loginConflict can return both, ErrUserAlreadyExists or ConflictErr

Reply:
Why are those different?

Reply:
One error comes from the check for users existing. One is if the users email/login conflicts by casinsensitive. So yeas it's the same error essentially, but we do not know which error is being reported from. Does this make sense? 

Maybe wrapped error would be better here?

Reply:
trying to investigate a issue that a user has from the 4986 support ticket

Reply:
basically we are missing this error - https://github.com/grafana/grafana/blob/7a8f1ab566f6a56a2917e44a3da892c16504054c/pkg/services/user/userimpl/store.go#L307

Reply:
Ah, I understand 🙂